system/jhtree/jhinplace.cpp

```diff
@@ -2109,13 +2127,21 @@ bool CInplaceLeafWriteNode::add(offset_t pos, const void * _data, size32_t size,
                         //MORE: Some compressors do not support incremental compression.  For those we should continue to
                         //gather uncompressed, but may also need to move the buffer to the context so it is reused (or
                         //maybe swap with the context).
-                        gatherUncompressed = false;
+                        if (!ctx.options.recompress)
+                            gatherUncompressed = false;
                     }
                 }
                 // else compressed does not fit either - better to remain uncompressed.
             }
             else
             {
+                if (ctx.options.recompress)// && uncompressed.length() < nodeSize * 10)
```

Comment:
Should the comment be removed (or rephrased to indicate that it's a potential improvement, but that it's not in use because ???)

Reply:
Yes.  I'll remove (and add a comment).